cli/libreant_db.py

```diff
@@ -95,5 +96,93 @@ def export_all(pretty):
     click.echo(json.dumps(volumes, indent=indent))
 
 
+@libreant_db.command(name='append-file', help='uploads a file to an existing volume')
+@click.option('-i', '--volumeid', type=click.STRING, )
+@click.option('-f', '--filepath', type=click.Path(exists=True,resolve_path=True), multiple=True, help='the path to the media to be uploaded')
+@click.option('-n', '--name', type=click.STRING, multiple=True, help='[optional] name of the media including the extension')
+@click.option('-m', '--mime', type=click.STRING, multiple=True, help='[optional] mime type of the media')
+@click.option('-t', '--notes', type=click.STRING, multiple=True, help='[optional] notes about the media')
+def append_file(volumeid,filepath,name,mime,notes):
+    #attachments = map_attachments(filepath,name,mime,notes)
+    attachments = map_attachments(filename=list(filepath) if not filepath==None else [],\
+                                  name=list(name) if not name==None else [],\
+                                  mime=list(mime) if not mime==None else [],\
+                                  notes=list(notes) if not notes==None else [])
+    try:
+        arc.insert_attachments(volumeid,attachments)
+    except:
+        click.secho('An upload error occurred in updating an attachment!',fg='yellow', err=True)
+        exit(4)
+
+
+@libreant_db.command(name='insert-volume', help='uploads a volume to the db')
+@click.option('-l', '--language', type=click.STRING, help='specify the language of the media you are going to upload')
+@click.option('-f', '--filepath', type=click.Path(exists=True,resolve_path=True), multiple=True, help='the path to the media to be uploaded')
+@click.option('-n', '--name', type=click.STRING, multiple=True, help='[optional] name of the media including the extension')
+@click.option('-m', '--mime', type=click.STRING, multiple=True, help='[optional] mime type of the media')
+@click.option('-t', '--notes', type=click.STRING, multiple=True, help='[optional] notes about the media')
+@click.option('-x', '--extravalues', type=click.STRING, help='[optional] additional parameters')
```

Comment:
please use a coherent naming.
There are no "files" nor "media", only attachments.
I suppose that `extravalues` is just `volume metadata`. To understand this I had to read the code